source/whats-new.txt

```diff
@@ -21,6 +22,28 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.7.1:
+
+What's New in 4.7.1
```

Comment:
From what I understand about the current situation, there's a request to remove the versions of the drivers that exhibit this behavior from public access. If that's the case for Java 4.7 driver, could this PR replace the "What's New" entry for 4.7 with this one (4.7.1) and include an admonition mentioning it was removed (and perhaps why unless that's covered by other content)?

Reply:
Please correct me if I am mistaken, but I don't believe 4.7 can be removed (see this [spreadsheet](https://docs.google.com/spreadsheets/d/1XmuN7e9WnQVPalVSBCgiLpRPvgSIbgNyustR1nIcZy8/edit#gid=0) which links to this [article](https://central.sonatype.org/faq/can-i-change-a-component/)). I'll confirm this with a DBX-JVM team member before publishing this change however.

If 4.7 is removed, I agree your suggestions will improve the page, and I will implement them.

Does the plan to hold off on making this change and ask JVM team if 4.7 can be removed and then implement this change if it can be removed sound alright to you?

Reply:
Yes, I think this involves getting information from the Java DBX team. Regardless of whether it can be removed from sonatype, we can remove all mention and links to it in our docs. We've done something similar in the past for another driver a couple years ago.

Reply:
Artifacts generally can not be removed from Maven Central.  We have to live with that.